### PR TITLE
[Dessert] - Add API Intro Post

### PR DESCRIPTION
**Pull request by EchoTang04**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: docs/Dessert/Apps/2021-11-26-APIintro.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+template: overrides/blogs.html
+---
+
+# 有趣的 API 介绍
+
+!!! info
+    作者：Echo，发布于 2021-11-26，阅读时间：约 3 分钟，微信公众号文章链接：[:fontawesome-solid-link:]()
```
Comment (Tina0330) [inline docs/Dessert/Apps/2021-11-26-APIintro.md:8]: 糖糖，这个要复制以前的模板，阅读时间是script自己加的，但要注意模板的格式。

File: docs/Dessert/Apps/2021-11-26-APIintro.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+<figure>
+  <img src="https://cdn.jsdelivr.net/gh/BulletTech2021/Pics/img/boredAPI.png"/>
+  <figcaption>活动参数参考</figcaption>
+</figure>
+
+
+## 3 代码实现
+
+API（Application Programming Interface）是应用程序接口。让我们不用重复造轮子，可以只使用简单的几行代码实现各种有趣的功能。在以下 demo 中，我们分别调用了两个API来随机生成土味情话和活动推荐。你要做的就是重新run一遍。代码如下：
```
Comment (Tina0330) [inline docs/Dessert/Apps/2021-11-26-APIintro.md:46]: 英文和中文之间 之前都是没有空格的，也可以去一下，还有后面的

File: docs/Dessert/Apps/2021-11-26-APIintro.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+# 累了，听听今日份彩虹屁
+sweet_words = []
+url1 = 'https://chp.shadiao.app/api.php'
+for i in range(5):
+    time.sleep(random.uniform(0.1,1))
+    response1 = requests.get(url1, headers=getHeaders())
+    sweet_words.append(response1.text)
+truth = pd.DataFrame(sweet_words,columns=['别看了说的就是你'])
+truth
```
Comment (Tina0330) [inline docs/Dessert/Apps/2021-11-26-APIintro.md:89]: 很好奇出来的结果，可以分享吗，还有那个to-do

**Review by Tina0330 — APPROVED** [review]

File: docs/Dessert/Apps/2021-11-26-APIintro.md (hunk 3 above)
Comment (EchoTang04) [reply docs/Dessert/Apps/2021-11-26-APIintro.md:89]: 结果就是上面的示例图。因为想先展示结果，就把code放在后面啦。
@Tina0330

File: docs/Dessert/Apps/2021-11-26-APIintro.md (hunk 1 above)
Comment (EchoTang04) [reply docs/Dessert/Apps/2021-11-26-APIintro.md:8]: 我复制了以前的文章的模板，这样可以吗？

File: docs/Dessert/Apps/2021-11-26-APIintro.md [hunk 4]
```diff
@@ -0,0 +1,115 @@
...
+    todo.append((item["activity"],item["type"],item["participants"],item["price"],item["link"],item["accessibility"]))
+todo_list = pd.DataFrame(todo,columns=['事项','类别','参与人数','价格(0-1)','链接','可行性(0-1)'])
+todo_list['可行性(0-1)'] = 1-todo_list['可行性(0-1)'] 
+todo_list
+
+```
+
+## 4 小结
+
+今天介绍的两个API都非常容易上手，因为它不需要网络令牌或 API 密钥，没有注册时隐私泄露和广告的烦恼。只要你有python的环境，无脑运行一下code，可以每次都获得不一样的信息哦。希望能给大家带来一点笑容和欢乐。那么以上，祝大家周末愉快！
```
Comment (Realvincentyuan) [inline docs/Dessert/Apps/2021-11-26-APIintro.md:111]: Python，P要大写

**Review by Realvincentyuan — APPROVED** [review]
哈哈

File: docs/Dessert/Apps/2021-11-26-APIintro.md (hunk 2 above)
Comment (EchoTang04) [reply docs/Dessert/Apps/2021-11-26-APIintro.md:46]: 已修改，谢谢Tina! @Tina0330

Comment (EchoTang04) [reply docs/Dessert/Apps/2021-11-26-APIintro.md:46]: 已修改，谢谢Tina! @Tina0330

File: docs/Dessert/Apps/2021-11-26-APIintro.md (hunk 4 above)
Comment (EchoTang04) [reply docs/Dessert/Apps/2021-11-26-APIintro.md:111]: 好的！已修改~谢谢袁老板！

File: docs/Dessert/Apps/2021-11-26-APIintro.md (hunk 1 above)
Comment (Tina0330) [reply docs/Dessert/Apps/2021-11-26-APIintro.md]: 可以的！

**Review by jhqian0909 — APPROVED** [review]

**Comment by EchoTang04** [comment]
update time from Friday to Saturday.
@Realvincentyuan @jhqian0909 @Tina0330 @jz263

**Pull request merged**